### PR TITLE
allow additional options to be passed in

### PR DESCRIPTION
**Pull request by grawk**
to middleware method

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ module.exports = {
         // even though there's no reason javascript URLs have to have .js in them.
         return mlpp.languagePackPath(locale, true);
     },
-    middleware: function () {
-        return mlpp.middleware({nosuffix: true});
+    middleware: function (_options) {
+        var options = Object.assign({nosuffix: true}, _options);
```
Comment (gabrielcsapo) [inline index.js:15]: 👍

**Review by gabrielcsapo — APPROVED** [review]

**Comment by aredridel** [comment]
Ooh, what's the use case for this?

**Comment by grawk** [comment]
@aredridel The use case is to pass a custom locale object key to `makara-languagepackpath`:
https://github.com/krakenjs/makara-languagepackpath/pull/1

The high level is that we want to pass in a bcp47-like object to avoid the bcp47 parse step in languagepackpath. Otherwise, the locale string the rest of the application is using (let's say.. `zh-C2`) would end up throwing an error here.

Another associated change:
https://github.com/krakenjs/dust-makara-helpers/pull/14

**Comment by aredridel** [comment]
Makes some sense, but why pass around what ends up being a mostly internal representation?

**Comment by gabrielcsapo** [comment]
going to merge @aredridel if you need context I can fill you in on that, this is time sensitive 🕐

**Comment by aredridel** [comment]
Cool :)

**Pull request merged**